Gemfile

```diff
@@ -3,6 +3,10 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in hydra-remote_identifier.gemspec
 gemspec
 
+gem 'cirneco'
+gem 'byebug'
+gem 'dotenv'
```

Comment:
Is dotenv needed in the gem?

Reply:
We ended up not needing cirneco, byebug or dotenv in the Gemfile.